### PR TITLE
DocDB Cluster Instance Certificate Rotated

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/aws/documentDB/docdbCertificateRotated.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'DocumentDB Cluster Instance Certificate Rotation',
+    category: 'DocumentDB',
+    domain: 'Databases',
+    severity: 'High',
+    description: 'Ensure that AWS DocumentDB cluster instance certificates are rotated.',
+    more_info: 'Enabling the AWS DocumentDB cluster certificate rotation ensures that your cluster\'s TLS certificates are automatically rotated to maintain security and compliance standards. This feature helps in seamlessly updating certificates without downtime, ensuring continuous protection for data in transit within the DocumentDB cluster.',
```
Comment (alphadev4) [inline plugins/aws/documentDB/docdbCertificateRotated.js:10]:
```suggestion
    more_info: 'AWS DocumentDB cluster certificate rotation ensures that cluster\'s TLS certificates are automatically rotated to maintain security and compliance standards. This feature helps in seamlessly updating certificates without downtime, ensuring continuous protection for data in transit within the DocumentDB cluster.',
```

File: plugins/aws/documentDB/docdbCertificateRotated.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    recommended_action: 'Modify DocumentDB cluster instance and rotate the old server certificate.',
+    link: 'https://docs.aws.amazon.com/documentdb/latest/developerguide/ca_cert_rotation.html',
+    apis: ['RDS:describeDBInstances'],
```
Comment (alphadev4) [inline plugins/aws/documentDB/docdbCertificateRotated.js:13]: why are we not using the DocDB:describeDBClusters call?

**Review by alphadev4 — CHANGES_REQUESTED** [review]

File: plugins/aws/documentDB/docdbCertificateRotated.js (hunk 2 above)
Comment (muzzamilinovaqo) [reply plugins/aws/documentDB/docdbCertificateRotated.js:13]: We need to list the docdb instances here. And DocDb:describeDBInstances and rds:describeDBInstances, lists same data

Comment (alphadev4) [reply plugins/aws/documentDB/docdbCertificateRotated.js:13]: We need to list the docdb instances here. And DocDb:describeDBInstances and rds:describeDBInstances, lists same data

**Review by alphadev4 — APPROVED** [review]

**Pull request merged**